api/src/main/java/org/cardanofoundation/rosetta/api/account/service/AccountServiceImpl.java

```diff
@@ -67,9 +81,9 @@ public AccountCoinsResponse getAccountCoins(AccountCoinsRequest accountCoinsRequ
     }
     log.debug("[accountCoins] Address is Era");
     if (Objects.nonNull(currencies)) {
```

Comment:
Can be rewritten wit Optional.ofNullable(..)

Reply:
This will add complexity with almost no improvements. Let's keep it with a simple if() statement.
https://stackoverflow.com/questions/52038417/should-optional-ofnullable-be-used-for-null-check